index.org

```diff
@@ -270,7 +271,7 @@ For example, when executing =cargo rtic-scope --bin blinky [options]=:
    For example, =device = stm32f4::stm32f401=, =binds = SysTick=, and =binds = EXTI1= might be extracted.
    Here, each IRQ label is associated with the RTIC task it is bound to.
 
-   This parsing step places some restrictions on how the source code for an RTIC application can be written. Refer to [[RTIC application constrains]].
+   This parsing step places some restrictions on how the source code for an RTIC application can be written. Refer to [[RTIC application constraints]].
 3. An adhoc cdylib crate is then built which translates IRQ labels to IRQ numbers.
    ~libadhoc~ will look like the following with the parse information from the previous step:
```

Comment:
Where did *libadhoc* come from? Is that related to the cdylib crate?

Reply:
*libadhoc* is a cdylib (shared object file). The name is because I didn't really know what else to call it: its only purpose is to be built and loaded during setup and its content depends on the target application; its always built in an adhoc manner, hence the name. I'm open to renaming it.